Promact.Oauth.Server/src/Promact.Oauth.Server/wwwroot/app/users/user-add/user-add.component.ts

```diff
@@ -45,14 +46,14 @@ export class UserAddComponent {
     addUser(userModel) {
         this.loader.loader = true;
         userModel.JoiningDate = new Date(userModel.JoiningDate);
-        if (this.isSlackUserNameExist == true) {
-            if (this.isEmailExist == false) {
+        if (this.isSlackUserNameExist === false) {
```

Comment:
I will update code as below 
if (!this.isSlackUserNameExist) {
            if (!this.isEmailExist) {
                this.userService.registerUser(this.userModel).subscribe((result) => {
                    if (result) {
                        this.toast.show('User added successfully.');
                        this.redirectionRoute.navigate(['user/list']);
                    }
                    else if (!result) {
                        this.toast.show('User Name already exists.');
                    }
                    this.loader.loader = false;
                }, err => {
                });
            }
            else {
                this.loader.loader = false;
                this.toast.show('Email Address already exists.');
            }
        }